jwst/jump/tests/test_detect_jumps.py

```diff
@@ -82,27 +414,36 @@ def test_onecr_50_groups(setup_inputs):
     model1.data[0, 7, 5, 5] = 160.0
     model1.data[0, 8, 5, 5] = 170.0
     model1.data[0, 9, 5, 5] = 180.0
-    model1.data[0, 10:29, 5, 5] = 190.0
-    model1.data[0, 30:49, 5, 5] = 490.0
-    out_model = detect_jumps(model1, gain, rnModel, 4.0, False, 4.0)
-    assert (4 == np.max(out_model.groupdq[0, 5, 5, 5]))
-    outdqcr = out_model.groupdq[0, 5, 5, 5]
-    np.testing.assert_allclose(4, outdqcr)
+    print(np.arange(190, 290, 5))
+    model1.data[0, 10:30, 5, 5] = np.arange(190, 290, 5)
+    model1.data[0, 30:50, 5, 5] = np.arange(500, 600, 5)
+    out_model = detect_jumps(model1, gain, rnModel, 4.0, False, 4.0, 1, 200, 10, False)
+    assert (out_model.groupdq[0, 5, 5, 5] == 4) # CR in group 5
+    assert (out_model.groupdq[0, 30, 5, 5] == 4) # CR in group 30
+    assert (np.all(out_model.groupdq[0, 6:30, 5, 5] == 0)) # groups inbetween are not flagged
+
 
 # Need test for multi-ints near zero with positive and negative slopes
 
 @pytest.fixture
 def setup_inputs():
     def _setup(ngroups=10, readnoise=10, nints=1,
-                 nrows=1032, ncols=1024, nframes=1, grouptime=1.0, gain=1, deltatime=1):
+               nrows=1024, ncols=1032, nframes=1, grouptime=1.0, gain=1, deltatime=1,
+               gain_subarray = False, readnoise_subarray = False, subarray = False):
         times = np.array(list(range(ngroups)), dtype=np.float64) * deltatime
         gain = np.ones(shape=(nrows, ncols), dtype=np.float64) * gain
-        err = np.ones(shape=(nints, ngroups, nrows, ncols), dtype=np.float64)
-        data = np.zeros(shape=(nints, ngroups, nrows, ncols), dtype=np.float64)
+
+
         pixdq = np.zeros(shape=(nrows, ncols), dtype=np.float64)
```

Comment:
The data type for `pixdq` should be `np.uint32`.  This change should also be made in test_jump_step.py.

Reply:
changed it in all three test routines.